contracts/FlashLoanV1Router02.sol

```diff
@@ -0,0 +1,200 @@
+pragma solidity =0.6.6;
+
+import './interfaces/IFlashLoanV1Factory.sol';
+import './interfaces/IFlashLoanV1Pool.sol';
+import '@uniswap/lib/contracts/libraries/TransferHelper.sol';
+
+import "erc3156/contracts/interfaces/IERC3156FlashBorrower.sol";
+import "erc3156/contracts/interfaces/IERC3156FlashLender.sol";
+import './interfaces/IFlashLoanReceiver.sol';
+import './interfaces/IFlashLoanV1Router02.sol';
+import './libraries/FlashLoanV1Library.sol';
+import './libraries/SafeMath.sol';
+import './interfaces/IERC20.sol';
+import './interfaces/IWETH.sol';
+
+contract FlashLoanV1Router02 is IFlashLoanV1Router02, IERC3156FlashLender, IFlashLoanReceiver {
+    using SafeMath for uint;
+
+    // CONSTANTS
+    bytes32 public constant CALLBACK_SUCCESS = keccak256("ERC3156FlashBorrower.onFlashLoan");
+
+    // ACCESS CONTROL
+    // Only the `permissionedPairAddress` may call the `uniswapV2Call` function
+    address permissionedPoolAddress;
```

Comment:
What is the purpose of this?

Reply:
For access control, only the `permissionedPairAddress` may call the `executeOperation` function.